client/src/components/employer-dashboard-component/employer-dashboard-header/index.jsx

```diff
@@ -0,0 +1,94 @@
+import PropTypes from "prop-types";
+import { IconButton, Typography, Button } from "@mui/material";
+import AddIcon from "@mui/icons-material/Add";
+import ArrowBackIcon from "@mui/icons-material/ArrowBack";
+import { HeaderContainer } from "../index.styles";
+
+function EmployerDashboardHeader({ numEntries }) {
+  return (
+    <HeaderContainer>
+      <IconButton
+        sx={{
+          marginRight: 2,
+          marginLeft: 2,
+        }}
+        // onClick={}
+        size="small"
+      >
+        <ArrowBackIcon
+          sx={{
+            color: "gray",
+            cursor: "pointer",
+          }}
+        />
+      </IconButton>
+      <div style={{ flexGrow: 1 }}>
+        <Typography
+          style={{
+            fontFamily: "Arial",
+            fontSize: "34px",
+            fontWeight: 500,
+            lineHeight: "60px",
+            letterSpacing: "0.25px",
+            textAlign: "left",
+          }}
+        >
+          All Employers
+        </Typography>
+        <Typography
+          style={{
+            fontFamily: "Arial",
+            fontSize: "14px",
+            fontWeight: 500,
+            lineHeight: "0px",
+            letterSpacing: "0.1px",
+            textAlign: "left",
+            color: "#00000099",
+          }}
+        >
+          {numEntries} Employers
+        </Typography>
+      </div>
+      <Button
+        sx={{
+          marginLeft: "auto",
+          marginBottom: "30px",
+          marginTop: "30px",
+          backgroundColor: "#ffffff0",
+          color: "#3568E5",
+          border: 1,
+          "&:hover": {
+            backgroundColor: "#ffffff0",
+            color: "#3568E5",
+          },
+        }}
+        startIcon={<AddIcon />}
```

Comment:
Can we change this to the download like the job leads export button

Reply:
Just need to swap this to the download icon 
`import DownloadIcon from '@mui/icons-material/Download';`